REFERENCE.md

```diff
@@ -1710,3 +1709,4 @@ Valid values: %r{^\S+$}
 namevar
 
 The name of the vhost to add
+
```

Comment:
these whitespace changes are the result of running the generate strings task. I can revert if wanted, though probably worth leaving it if that's the expected generated output format.